src/learn/qml/MuseScore/Learn/internal/ClassesPage.qml

```diff
@@ -63,141 +64,159 @@ FocusScope {
         }
     }
 
-    Rectangle {
-        anchors.top: parent.top
-        anchors.left: parent.left
-        anchors.leftMargin: root.sideMargin
+    ScrollBar.vertical: StyledScrollBar {
+        parent: root.parent
+
+        anchors.top: root.top
+        anchors.bottom: root.bottom
         anchors.right: parent.right
-        anchors.rightMargin: root.sideMargin
 
-        height: authorInfo.height
+        visible: root.contentHeight > root.height
+        z: 1
+    }
 
-        radius: 12
+    header: Item {
+        height: authorInfoContainer.height
+        anchors.left: parent.left
+        anchors.right: parent.right
+        Rectangle {
+            id: authorInfoContainer
+            anchors.top: parent.top
+            anchors.left: parent.left
+            anchors.leftMargin: root.sideMargin
+            anchors.rightMargin: root.sideMargin
 
-        color: ui.theme.backgroundPrimaryColor
+            height: authorInfo.height
+            width: (parent.width > 1100) ? 1000 : parent.width - 96
```

Comment:
I'd replace `96` with `2 * root.sideMargin`, to make clear what it does.

Is there a reason for not writing `Math.min(1000, parent.width - root.sideMargin)`?